compiler/one-cmds/onelib/constant.py

```diff
@@ -49,6 +49,7 @@ class CONSTANT:
         'fuse_prelu',
         'fuse_gelu',
         'fuse_rsqrt',
+        'fuse_rope',
```

Comment:
@jinevening , is it OK to add to O1 ?

Reply:
This will cause an error in our backend compiler.

Let's include this option to O1 when our backend compiler is ready.